src/app/(proper_react)/redesign/(public)/LandingView.module.scss

```diff
@@ -242,3 +242,74 @@
     }
   }
 }
+
+.socialProofWrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: $spacing-2xl $spacing-md;
+  gap: $spacing-sm;
+  text-align: center;
+
+  .pressLogos {
+    padding-top: $layout-md;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: $spacing-lg;
+    align-items: center;
+    justify-content: center;
+
+    @media screen and (min-width: $content-md) {
+      flex-direction: row;
+      flex-wrap: nowrap;
+    }
+
+    .label {
+      flex-basis: 100%;
+      font: $text-body-sm;
+      text-transform: uppercase;
```

Comment:
https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#css-issues

Reply:
On the other hand: https://www.timdunklee.com/notes/use-css-text-transform-for-uppercase-letters-a11y/

Since it's not _vital_ that the text is rendered in uppercase, one workaround is to only uppercase it in English: https://github.com/mozilla/blurts-server/blob/c8dcdb67284d35d40d7ceb2479cb148b3d88a17c/src/app/(proper_react)/redesign/(public)/PlansTable.module.scss#L152-L164

Reply:
Oh nice!